_includes/cockroachcloud/ccloud/backup-types.md

```diff
@@ -0,0 +1,6 @@
+[//]: # (Cloud backup types)
+
+Cockroach Cloud clusters support two types of backups:
+
+- Managed-service backups: automated full and incremental backups daily stored on Cockroach Cloud storage 
+- {% if page.name == "run-bulk-operations.md" %} Self-service backups {% else %} [Self-service backups](../cockroachcloud/run-bulk-operations.html) {% endif %}: backups to your own storage buckets using the [`BACKUP`](../{{site.versions["stable"]}}/backup.html) statement
```

Comment:
Nice job on the if/else logic, well-placed. 

I discussed the naming of the backup types with Liv again, and she felt we should rename "self-service" to "customer-owned backups". If you could make that change here that would be great. We will share this with Liv after these docs changes to see if she has any feedback too.

On the content here, I might slightly adapt the description to match the changes to the other type, something like the following:

> - Customer-owned backups: You can take manual backups and store them in your cloud storage buckets using `BACKUP`.

With these changes, the party taking the backups is the subject of the sentence for both bullet points and so provides that further distinction. Great to link to the `BACKUP` page 👍 You could also link to the Use Cloud Storage for Bulk Operations on "cloud storage bucket".